pre-myget.bat

```diff
@@ -0,0 +1,2 @@
+call dnvm upgrade -r clr
```

Comment:
**NIT:** You should probably specify the version of the DNX runtime to install, it is not really a problem now because this command will not work for RC2 ever but in general.